crates/voicevox_core/src/voice_model.rs

```diff
@@ -99,6 +121,16 @@ impl VoiceModel {
         join_all(vvm_paths).await.into_iter().collect()
     }
     const ROOT_DIR_ENV_NAME: &str = "VV_MODELS_ROOT_DIR";
+
+    /// スタイルIDからモデル内IDを取得する。
+    /// モデル内IDのマッピングが存在しない場合はそのままスタイルIDを返す。
+    pub(crate) fn style_id_to_model_inner_id(&self, style_id: StyleId) -> ModelInnerId {
```

Comment:
ここの名前ですが、manifest.jsonのフィールドとして存在している`style_id_to_model_inner_id`からは離した方が混乱を避けられるかなと思いました。

```suggestion
    pub(crate) fn model_inner_id_for(&self, style_id: StyleId) -> ModelInnerId {
```

Reply:
metasとかも被ってるのであまり気にならないかも。
でも名前が短いのは良いことなので適用します！